python/tests/test_cf_naming_clarity.py

```
@@ -0,0 +1,162 @@
+            if re.search(r"best_score\s*\{", line) or "float best_score" in line:
+                if "free energy" in lower or "Δg" in lower or "delta g" in lower:
+                    offenders.append(f"{i}: {line.strip()}")
+            # Inline comment on the declaration line only (after //)
+            if "float best_score" in line and "//" in line:
+                comment = line.split("//", 1)[1].lower()
+                if "free energy" in comment or "Δg" in comment:
+                    if "not" not in comment and "≠" not in comment and "!=" not in comment:
+                        offenders.append(f"{i}: {line.strip()}")
+        assert not offenders, (
+            "best_score must be documented as CF/contact-function scoring proxy, "
+            f"not free energy:\n" + "\n".join(offenders)
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)